src/lib/extensions/extensionHost.ts

```
@@ -957,6 +978,68 @@ export class ExtensionHost {
+				x: number,
+				y: number,
+				size: number,
+				color: string,
+				weight: "thin" | "light" | "regular" | "bold" | "fill" = "regular",
+			): void {
+				const cacheKey = `${name}:${weight}`;
+				let path = host.iconPathCache.get(cacheKey);
+
+				if (!path) {
+					// 1. Get the Icon component from the project's library
+					const Icon = (PhosphorIcons as any)[name];
```

Biome (2.4.14)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)